plugwise/smile.py

```diff
@@ -47,46 +51,52 @@ def _all_device_data(self) -> None:
         """Helper-function for get_all_devices().
         Collect initial data for each device and add to self.gw_data and self.gw_devices.
         """
-        for device_id, device in self._devices.items():
-            bs_dict: dict[str, bool] = {}
-            s_dict: dict[str, Any] = {}
-            sw_dict: dict[str, bool] = {}
+        for item in self._appl_data:
```

Comment:
Same as before, in helper also spotted this (but not around line 1300 where we still have the old way)